packages/contracts/contracts/Unirep.sol

```diff
@@ -288,6 +293,7 @@ contract Unirep is IUnirep, VerifySignature {
             fieldIndex,
             change
         );
+        attestationCounter++;
```

Comment:
I think it's ok to increment this value for all attestations. Technically we only need to increment this value when an attestation is made to a replacement field. But we'll only hit 2^48 attestations after 9 years of 1 million attestations per second.

Plus i think it's interesting to have an onchain variable tracking the attestation count for the system.

Reply:
In this case, I'll leave it as it is